packages/suite/src/views/wallet/coinmarket/common/CoinmarketForm/CoinmarketFormOfferFiatAmount.tsx

```diff
@@ -28,7 +28,7 @@ interface CoinmarketFormOfferFiatAmountProps {
 }
 
 export const CoinmarketFormOfferFiatAmount = ({ amount }: CoinmarketFormOfferFiatAmountProps) => {
-    const formattedAmount = amount?.replace(/\B(?=(\d{3})+(?!\d))/g, ' ') ?? '';
+    const formattedAmount = amount ? new Intl.NumberFormat().format(Number(amount)) : '';
```

Comment:
Please provide user's `locale`
`new Intl.NumberFormat(locale)` check the occurences in codebase

Reply:
Fixed